tools/annotation_tools/template_tool/frontend/src/spec/action_dict_components.schema.json

```diff
@@ -0,0 +1 @@
+../../../../../../base_agent/documents/json_schema/action_dict_components.schema.json
```

Comment:
Should we use the path from root instead ?

Reply:
this is a relative path symlink. It's not pretty the alternative with absolute paths is more likely to introduce path resolution bugs, imo.